### DISCUSSION TITLE
Hosting Mozilla Hub on Azure

### DISCUSSION THREAD

**Discussion by michaelcsso** [mozilla/hubs#5734] [Hubs Cloud Questions] [answered]
May I know is there anyone did try to host the Mozilla hubs on Azure instead of Hubs Cloud?  Would it be very difficult to do so?

**Comment by matthewbcool** [comment, marked as answer]
Certainly possible but I don't know of anyone that has figured out and documented this specific pathway yet. I recommend joining our dev channels in discord to search for more information if you wish to pursue this further. https://discord.gg/YhcsU9JY
